docs/setup/configuration.md

```diff
@@ -22,15 +22,17 @@ The `server` configuration is required when you're running the `server/server.js
 ```json
 {
     "server": {
-        "port": 9999
+        "port": 9999,
+        "personalAccessTokenGitLab": ""
```

Comment:
Because it is exactly that: either a personal access token for GitLab, or a GitLab personal access token. I like to make variables descriptively of what the are.

Fine by me to leave it at this and accept this MR anyway, it's just a matter of personal preference I guess.

Reply:
@eXistenZNL I 100% agree on descriptive variable names, but in my head `personalAccessTokenGitLab` and `personalAccessTokenForGitLab` are the same thing :innocent: